arangodb-net-standard/AqlFunctionApi/Models/PostExplainAqlQueryResponseIndex.cs

```diff
@@ -1,19 +1,68 @@
-﻿using System;
+﻿using ArangoDBNetStandard.IndexApi.Models;
 using System.Collections.Generic;
-using System.Text;
 
 namespace ArangoDBNetStandard.AqlFunctionApi.Models
 {
     public class PostExplainAqlQueryResponseIndex
```

Comment:
I think there is an oppotunity to extract the properties common to `PostExplainAqlQueryResponseIndex` and `IndexResponseBase` into an interface. I will take a closer look later and open a separate pull request if it makes sense.

Reply:
It makes sense to do it if it's possible.